client/js/utils.js

```diff
@@ -48,6 +49,15 @@ function isOpInChannel(channel) {
 	return isOP;
```

Comment:
As a side note, this only works for "op", doesn't work for "halfop" "owner", "admin" or anything else. So literally it tells you if a user is op, if the user is more privileged than op, it'll return false.

Reply:
Sure, but we currently have some support for them, so we should probably deal with them in this somehow.